.github/workflows/sync-flow-go.yml

```diff
@@ -45,6 +47,16 @@ jobs:
           NEW_BRANCH=auto-cadence-upgrade/$( date +%s )/${{ github.event.pull_request.head.ref }}
           echo "NEW_BRANCH=$NEW_BRANCH" >> $GITHUB_ENV
 
+      # create feature branch if needed
+      - name: Create feature-secure-cadence branch
```

Comment:
@robert-e-davidson3 Shouldn't this match with the above branch name (`feature/secure-cadence`)?